handler/src/main/java/io/netty/handler/ssl/JdkAlpnSslEngine.java

```diff
@@ -39,7 +40,8 @@ static boolean isAvailable() {
     }
 
     private static void updateAvailability() {
-        if (available) {
+        // TODO: Add support for ALPN when using Java9 and still be able to configure it the Netty way.
```

Comment:
Any updates on this? I've opened https://github.com/netty/netty/issues/6933 to track ALPN support.